app/assets/stylesheets/common/base/topic-post.scss

```diff
@@ -1633,20 +1633,21 @@ span.mention {
 
 .open-popup-link {
   position: sticky;
-  left: 0.5rem;
-  top: 0.5rem;
+  left: 1rem;
   opacity: 0%;
   white-space: nowrap;
-  display: block;
 }
 
 .fullscreen-table-wrapper {
   transition: all 0.6s cubic-bezier(0.165, 0.84, 0.44, 1);
   display: block;
-}
+  position: relative;
 
-.expand-table-icon {
-  margin-right: 4px;
+  &-buttons {
+    position: absolute;
+    top: 0.5rem;
+    left: 0.5rem;
```

Comment:
<img width="719" alt="image" src="https://user-images.githubusercontent.com/368961/227594650-a3c78392-c5aa-4a69-8975-cb7bc941d847.png">

I'm ok with this, but since we're now overlaying these buttons, I wonder whether in a followup PR we should switch to using icon-only buttons. It would take up a lot less space (and I think most users can still easily tell what the buttons do by just seeing icons).